components/Accordion/index.tsx

```diff
@@ -8,9 +8,9 @@ const Accordion = ({ children, icon, iconColor, title }) => {
   return (
     <details className="Accordion" onToggle={() => setIsOpen(!isOpen)}>
       <summary className="Accordion-title">
-       <i className={`Accordion-toggleIcon${isOpen ? '_opened' : ''} fa fa-chevron-right`}></i>
-       {icon && <i className={`Accordion-icon fa ${icon}`} style={{ color: `${iconColor}` }}></i>}
-       {title}
+        <i className={`Accordion-toggleIcon${isOpen ? '_opened' : ''} <i class="fa fa-regular fa-chevron-right"></i>`}></i>
+        {icon && <i className={`Accordion-icon fa-duotone fa-solid ${icon}`} style={{ color: `${iconColor}` }}></i>}
```

Comment:
ugh, laaaaaame that we now have to specify an extra `.fa-style` class all the time now!

Reply:
yeaaaah I hate it. it should be backwards compatible with the regular style if people don’t specify it, but I would assume FA would drop support eventually since it’s not how they document it